common/tests/integration_tests.rs

```diff
@@ -128,12 +129,201 @@ fn basic2() {
         .collect::<Vec<_>>();
     csv.verify_last_header_finalization(&fp).unwrap();
     light_client.update(block_header, fp).unwrap();
+    let commits = csv.get_total_commits();
     let merkle_tree = OneshotMerkleTree::create(
-        csv.get_total_commits()[1..=3]
+        commits[1..=(commits.len() - 2)]
             .iter()
             .map(|c| c.to_hash256())
             .collect(),
     );
     let merkle_proof = merkle_tree.create_merkle_proof(tx.to_hash256()).unwrap();
     assert!(light_client.verify_transaction_commitment(&tx, 1, merkle_proof));
 }
+
+#[test]
+fn basic3() {
+    let member_number = 10;
+    let (rs, keys) = test_utils::generate_standard_genesis(member_number);
+    let genesis_info = rs.genesis_info.clone();
+    let genesis_header = rs.genesis_info.header.clone();
+
+    let (height, timestamp) = (1, 0);
+    let mut csv = CommitSequenceVerifier::new(genesis_header.clone(), rs.clone()).unwrap();
+    let mut light_client = LightClient::new(genesis_header);
+    let author = &keys[1];
```

Comment:
Ditto. How about just using `rs.query_name(&keys[0].0).unwrap()` as already existing tests?

Reply:
Well, that's actually what we're doing below.
https://github.com/postech-dao/simperby/pull/366/files/a656689b138acc7663b4b0def1c865cfaeb94274#diff-285c633c8231379e5878f13afd40fadc13ffcb5390b67cf93aef7e3bedf2e917R166

Reply:
I know, but I think `author.0` is not that intuitive and the field `author` in `Transaction` is overlapping so I thought just keeping the way as already existing tests have better readability.

Reply:
fixed